assets/src/edit-story/components/checklist/checks/videoOptimization.js

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { _n, __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { useCallback, useMemo, useRef } from 'react';
+import { useFeature } from 'flagged';
+import { useLocalMedia, useStory } from '../../../app';
+import { VIDEO_SIZE_THRESHOLD } from '../../../app/media/utils/useFFmpeg';
```

Comment:
FYI this will change in https://github.com/google/web-stories-wp/pull/7528/files#diff-b6be8a8656fd7a2950908afb601c88095fe753f41ceaac1cdbc3984874940930R24-R25